README.md

```diff
@@ -88,7 +88,7 @@ of the course in the `book/` directory. Note that you have to separately build
 and zip exercises and add them to `book/html`. To build any of the translated
 versions of the course, run `MDBOOK_BOOK__LANGUAGE=xx mdbook build -d book/xx`
 where `xx` is the ISO 639 language code (e.g. `da` for the Danish translation).
-[TRANSLATIONS.md](TRANSLATIONS.md) contains further instructions.
+For complete info about installing necessary tools , refer [CONTRIBUTING.md](CONTRIBUTING.md#formatting-and-preparation)
```

Comment:
This part is really about the translation infrastructure, it's not about how to auto-format the files in the course. So I think it should stay as it is.

But the installation instructions above should move to the CONTRIBUTING guide.

Reply:
okay,
I have done done as per your suggestion in the latest commit 

Please check this out 
[Fixed Readme.md , Translation.md](https://github.com/google/comprehensive-rust/pull/1431/commits/43e89f2d05b0e5cf4e90982689ff4676986345cd)

Reply:
Cool, much better! Please also move the content of the `## Building` section to the contribution guide. Then let the README send everybody to the CONTRIBUTING file where all information about installing the course should live.

Reply:
Sorry, I thought more about it and I think it's good to split it the way you have it now. I updated it a tiny bit directly and the PR should be merged in a few minutes.